.jshintrc

```diff
@@ -13,5 +13,14 @@
   "quotmark": "single",
   "undef": true,
   "unused": true,
-  "strict": true
+  "strict": true,
+  "globals"   : {
+    /* MOCHA */
```

Comment:
@sindresorhus I didn't know about that. I've modified the pull request to use `"mocha" : true`